### PR TITLE
Remove one step connecting one another.

### PR DESCRIPTION
**Pull request by minsulee2**
Updates:
- Now each node **directly** asks bi-connection via peerInfo.
- Leave TODO comments for next works.

To avoid too big PR, Small updates just has done. I am also working on with AIM-021! No worries!

### CODE REVIEW

File: p2p/index.js [hunk 1]
```diff
@@ -312,6 +298,7 @@ class P2pClient {
   sendAddress(socket) {
```
Comment (platfowner) [inline p2p/index.js:298]: What about giving a new name e.g. sendPeerInfo() as it's doing so now?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
**Please make sure the code is kept working and the tests pass all the time.**
Thanks!

File: p2p/index.js (hunk 1 above)
Comment (minsulee2) [reply p2p/index.js]: Updated! But it will be renamed again as it is going to do more than that in the next update!

**Comment by minsulee2** [comment]
@platfowner Thanks for the fast review! Before moving on to the next updates, I will check some tests first!!

**Pull request merged**